src/status_im2/contexts/wallet/events.cljs

```diff
@@ -9,8 +9,10 @@
     [status-im2.common.data-store.wallet :as data-store]
     [status-im2.contexts.wallet.temp :as temp]
     [taoensso.timbre :as log]
```

Comment:
how about we move events for send stuff to `status-im2.contexts.wallet.send.events` - there's quite a lot there and this file is gonna get quite large 👍